src/databaseTests/src/test/kotlin/org/vaccineimpact/api/databaseTests/tests/ScenarioTests.kt

```diff
@@ -48,6 +51,107 @@ class ScenarioTests : RepositoryTests<ScenarioRepository>()
         }
     }
 
+    @Test
+    fun `getScenariosForTouchstone returns ordered scenarios`()
+    {
+        val disease1 = "disease-abc"
+        val disease2 = "disease-def"
+        withDatabase {
+            setUp(it)
+            setUpDisease(it, disease2)
```

Comment:
I know you didn't introduce this hear, but this seems like a good time to improve the name of this function. e.g. `createDiseaseWithThreeScenarios` or something. Otherwise these tests aren't very readable.